src/app/app.module.ts

```diff
@@ -32,6 +33,7 @@ import { TruncateModalComponent } from './tables/truncate-modal/truncate-modal.c
 import { TestsTableComponent } from './tables/tests-table/tests-table.component';
 import { TestResultsComponent } from './story/test-results/test-results.component';
 import { HTTPErrorHandler } from './interceptors/http-error-handler';
+import { TabsModule } from 'ngx-bootstrap/tabs';
```

Comment:
Can you move this up in the previous section of imports? I'm trying to keep third-party packages in the first section and local imports in the second section.